server.js

```diff
@@ -0,0 +1,59 @@
+const http = require('http');
+const fs = require('fs');
+//const public = require('./public')
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMeth = request.method;
+console.log(uri)
+  function reqResp(uri, type) {
+    fs.readFile(`./public${uri}.html`, (err, data) => {
+      if (err) throw 'error';
+      response.setHeader('Content-Type', `text/${type}`)
+      response.writeHead('200', 'OK')
+      response.write(data)
+      response.end();
+    })
+  }
+
```

Comment:
You will want your switch case to switch on a know set of variable, like method. We know we will only need to handle `GET`, `POST`, `PUT`, and `DELETE`, this is perfect for a switch. The `uri` can contain anything so we need the code to be more dynamic we responding to it.